impeller/renderer/pipeline_descriptor.h

```diff
@@ -115,6 +115,10 @@ class PipelineDescriptor final : public Comparable<PipelineDescriptor> {
 
   WindingOrder GetWindingOrder() const;
 
+  void SetPrimitiveType(PrimitiveType type);
```

Comment:
The setters return a reference to the descriptor so we can chain them. Though I don't see that pattern followed a lot.